configs/trainers/image_vae.yaml

```
@@ -9,5 +9,9 @@
     _partial_: true
     lr: 0.001
   device: ${devices.0}
+  logger:
+    _target_: ami.tensorboard_loggers.StepIntervalLogger
+    log_dir: ${paths.log_dir}/tensorboard
```

`ami/configs/paths/default.yaml`に`tensorboard_dir: ${paths.log_dir}/tensorboard`を追加してください。そして以下のようにすると良いと思います。Trainerごとに個別のTensorBoardディレクトリを用意する理由は、再開した際にその個別フォルダに新規のログファイルが生成され、引継が行われるというTensorBoardの仕様のためです。

```suggestion
    log_dir: ${paths.tensorboard_dir}/image_vae_trainer
```

---

https://github.com/MLShukai/ami/pull/158/commits/da4036c8674ea4183902e5d9503779873220a489
修正しました